tasks/internal_reconcile_report_phantom/README.md

```diff
@@ -0,0 +1,19 @@
+[![Known Vulnerabilities](https://snyk.io/test/github/nasa/cumulus-orca/badge.svg?targetFile=tasks/internal_reconcile_report_phantom/requirements.txt)](https://snyk.io/test/github/nasa/cumulus-orca?targetFile=tasks/internal_reconcile_report_phantom/requirements.txt)
+
+**Lambda function internal_reconcile_report_phantom **
+
+Visit the [Developer Guide](https://nasa.github.io/cumulus-orca/docs/developer/development-guide/code/contrib-code-intro) for information on environment setup and testing.
+
+- [Deployment](#deployment)
+- [pydoc internal_reconcile_report_phantom](#pydoc)
+
+<a name="deployment"></a>
+## Deployment
+```
+    see /bin/build_tasks.sh to build the zip file. Upload the zip file to AWS.
+```
+<a name="pydoc"></a>
+## pydoc internal_reconcile_report_phantom
```

Comment:
See get_current_archive_list for a new version of this.

Reply:
updated